### PR TITLE
refactor(swap): validate env vars

### PR DESCRIPTION
**Pull request by acdibble**

### CODE REVIEW

**Review by niklasnatter — APPROVED** [review]
🙌

File: packages/swap/src/event-handlers/swapEgressScheduled.ts [hunk 1]
```diff
@@ -22,15 +23,15 @@ const environmentByBlockHashCache = new CacheMap<
 >(60_000);
 
 const methodNotFoundRegExp = /Exported method .+ is not found/;
-const rpcConfig = { rpcUrl: process.env.RPC_NODE_HTTP_URL as string };
+const rpcConfig = { rpcUrl: env.RPC_NODE_HTTP_URL as string };
```
Comment (GabrielBuragev) [inline packages/swap/src/event-handlers/swapEgressScheduled.ts:26]:
```suggestion
const rpcConfig = { rpcUrl: env.RPC_NODE_HTTP_URL };
```

**Review by GabrielBuragev — APPROVED** [review]
image

File: packages/swap/src/event-handlers/__tests__/__snapshots__/batchSwap.test.ts.snap [hunk 2]
```diff
@@ -44,7 +44,7 @@ exports[`batch swap flow handles all the events: fee 0 1`] = `
 
 exports[`batch swap flow handles all the events: fee 1 1`] = `
 {
-  "amount": "0",
+  "amount": "990109",
```
Comment (acdibble) [inline packages/swap/src/event-handlers/__tests__/__snapshots__/batchSwap.test.ts.snap:47]: we now actually have a network set and hence a network fee

**Pull request merged**